exercises/isogram/src/example.erl

```diff
@@ -0,0 +1,24 @@
+-module(example).
+
+-export([is_isogram/1, test_version/0]).
+
+is_isogram(String) ->
+    check_isogram(string:to_lower(String)).
+
+check_isogram("") ->
+    true;
+check_isogram([H|T]) ->
+    B = is_alpha(H) andalso is_found_in(H, T),
+    not B andalso check_isogram(T).
+
+is_alpha(C) when ($a =< C), (C =< $z) ->
+    C /= string:to_upper(C);
```

Comment:
This will always return `true` and warm up your room ;) Can you literally return `true` please?

Reply:
Yes, of course you are right.

In is_isogram/1_ I convert to lower case, which is good practice in other languages as it generally avoids repetitive calculation of the same result.  In Erlang, this forced me to introduce a second routine, _check_isogram/1_ to do the recursion.

Your sensible suggestions about guards appear to have added repetitive calculation back in, which I expect increases both processor and memory foot print.

So I am now thinking I want to strip non-alphabetic characters from the input in _is_isogram/1_.  I guess I can use a guard on a list comprehension to do that.  Erlang has such nice features.

I'll be back.